docker-compose.yml

```diff
@@ -11,7 +11,23 @@ services:
       POSTGRES_PASSWORD: 'root'
       POSTGRES_DB: 'main-db'
     volumes:
-      - 'db-data:/var/lib/postgresql/data/'
+      - './data/postgres:/var/lib/postgresql/data/'
```

Comment:
I would like to have everything in one place, but I will remove it because it isn't essential for this PR and I don't
know how solve codecove errors 😃

Reply:
I like this change, but I don't know too :/ Let's postpone that.